### ISSUE TITLE
KeepaliveInterval multiplication

### ISSUE THREAD

**Issue by wuhkuh** [mcollina/aedes#72]
Good afternoon,

as I was wondering how you implemented the keepalive functionality I noticed this piece of code:

```
  if (packet.keepalive > 0) {
    client._keepaliveInterval = packet.keepalive * 1501
    client._keepaliveTimer = retimer(function keepaliveTimeout () {
      client.broker.emit('keepaliveTimeout', client)
      client.emit('error', new Error('keep alive timeout'))
    }, client._keepaliveInterval)
  }
```
after some thinking what the 1501 factor was, I came up with this.
1.5 (maximum keepalive until disconnect) * 1000 (milliseconds) + 1 (exceed 1 and get disconnected)

The problem with the current implementation is that if a client has a huge keepalive sent with the connect packet, quote: 

> The maximum keep alive is 18h 12min 15 sec

 it gets multiplied by 1501 instead of 1500 with an addition of one millisecond.
Do you want me to send a PR, or have I interpreted this code in a wrong manner and is this intended behavior?

IMO, this would be the fix:
`    client._keepaliveInterval = packet.keepalive * 1500 + 1`

**Comment by mcollina** [comment]
@wuhkuh please do, you are right!

**Comment by wuhkuh** [comment]
Fixed in #73
